chapters/memory.txt

```diff
@@ -3294,6 +3458,86 @@ include::{generated}/api/protos/vkUnmapMemory.txt[]
 include::{generated}/validity/protos/vkUnmapMemory.txt[]
 --
 
+ifdef::VK_EXT_map_memory2[]
+[open,refpage='vkUnmapMemory2EXT',desc='Unmap a previously mapped memory object',type='protos']
+--
+Alternatively, to unmap a memory object once host access to it is no longer
+needed by the application, call:
+
+include::{generated}/api/protos/vkUnmapMemory2EXT.txt[]
+
+  * pname:device is the logical device that owns the memory.
+  * pname:pMemoryUnmapInfo is a pointer to a slink:VkMemoryUnmapInfoEXT
+    structure describing parameters of the unmap.
+
+This function behaves identically to flink:vkUnmapMemory except that it gets
+its parameters via an extensible structure pointer rather than directly as
+function arguments.
+
+include::{generated}/validity/protos/vkUnmapMemory2EXT.txt[]
+--
+
+[open,refpage='VkMemoryUnmapInfoEXT',desc='Structure containing parameters of a memory unmap operation',type='structs']
+--
+The sname:VkMemoryUnmapInfoEXT structure is defined as:
+
+include::{generated}/api/structs/VkMemoryUnmapInfoEXT.txt[]
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to a structure extending this
+    structure.
+  * pname:memory is the slink:VkDeviceMemory object to be unmapped.
+
+.Valid Usage
+****
+  * pname:memory must: be currently host mapped
```

Comment:
Just to check - this is expected to allow unmapping of memory mapped with VkMapMemory2EXT (and vice-versa), right?

Reply:
Yes.  There's no reason why you can't mix+match.

Reply:
Ok, don't think this needs calling out, just wanted to make sure it was expected.

Reply:
(though this might need testing in the CTS test plan...)